bitsail-connectors/connector-doris/src/main/resources/doris-type-converter.yaml

```diff
@@ -59,6 +59,9 @@ engine.type.to.bitsail.type.converter:
   - source.type: date
     target.type: date.date
 
+  - source.type: bit
```

Comment:
does doris has `bit` datatype? I didn't see this type when I look at doris documentation.